bedrock/mozorg/templates/mozorg/technology/index.html

```diff
@@ -0,0 +1,255 @@
+          <li><a rel="external" href="https://aframe.io/blog/">{{ _('Visit the blog') }}</a></li>
+        </ul>
+      </footer>
+    </section>
+
+    <section id="gaming" class="even">
+      <div class="container">
+        <div class="inner-container">
+          <h2>{{ _('Using the Web to change the game') }}</h2>
+          <p>{{ _('With powerful Web technologies, pioneered by Mozilla, developers are pushing games to a new level.') }}</p>
+          <a rel="external" href="https://www.openwebgames.com/#/demos.html" class="button">{{ _('Play some demos') }}</a>
+        </div>
+      </div>
+      <footer>
+        <ul>
+          <li><a rel="external" href="https://games.mozilla.org/">{{ _('Learn more') }}</a></li>
+          <li><a rel="external" href="https://developer.mozilla.org/docs/Games">{{ _('Start building') }}</a></li>
+        </ul>
+      </footer>
+    </section>
+
+    <section id="iot" class="odd">
+      <div class="container">
+        <div class="inner-container">
+          <h2>{{ _('Adding trust to the Internet of Things') }}</h2>
+          <p>{{ _('Through open innovation, we’re bringing trust and transparency to networks of smart devices.') }}</p>
+          <a rel="external" href="https://connected.mozilla.org/2016/08/26/the-project-sensorweb-poster-experiment/" class="button">{{ _('Learn more') }}</a>
+        </div>
+      </div>
+      <footer>
+        <ul>
+          <li><a rel="external" href="https://wiki.mozilla.org/Connected_Devices">{{ _('Visit the wiki') }}</a></li>
+          <li><a rel="external" href="https://wiki.mozilla.org/Connected_Devices/Participation">{{ _('Start contributing') }}</a></li>
+        </ul>
+      </footer>
+    </section>
+
+    <section id="context-graph" class="even">
+      <div class="container">
+        <div class="inner-container">
+          <h2>{{ _('Building a browser that gets you') }}</h2>
+          <p>{{ _('The next generation of Web discovery is a browser that is more intuitive, useful and in tune with you.') }}</p>
+          <a rel="external" href="https://medium.com/@osunick/context-graph-its-time-to-bring-context-back-to-the-web-a7542fe45cf3#.76ofcthq2" class="button">{{ _('Learn more') }}</a>
+        </div>
+      </div>
+      <footer>
+        <ul>
+          <li><a rel="external" href="https://wiki.mozilla.org/Context_Graph">{{ _('Visit the wiki') }}</a></li>
+          <li><a rel="external" href="https://testpilot.firefox.com/experiments/activity-stream">{{ _('Try it now') }}</a></li>
+        </ul>
+      </footer>
+    </section>
+
+    <section id="rust" class="odd">
+      <div class="container">
+        <div class="inner-container">
+          <h2>{{ _('Inventing a safer programming language') }}</h2>
+          <p>{{ _('Supported by Mozilla, Rust allows browsers, systems and more to run much faster and more safely.') }}</p>
+          <a rel="external" href="#" class="button">{{ _('Learn more') }}</a>
```

Comment:
Proposed link as the blog entry isn't ready: https://developer.mozilla.org/docs/Mozilla/Rust